CHANGELOG.md

```diff
@@ -1,47 +1,37 @@
+- rkt's tar package didn't prefix the destination file correctly when using hard links in images. This was not a issue in rkt itself but was causing acbuild to misbehave ([#1852](https://github.com/coreos/rkt/pull/1852)).
+- ACIs with multiple dependencies can end up depending on the same base image through multiple paths. In some of those configuration with multiple dependencies, fetching the image via image discovery was not working. This is fixed and a new test ensures it will keep working ([#1822](https://github.com/coreos/rkt/pull/1822)).
+- The pod cgroups were misconfigured when systemd-devel is not installed. This was causing per-app CPU and memory isolators to be ineffective on those systems. This is now fixed but will require an additional fix for NixOS ([#1873](https://github.com/coreos/rkt/pull/1873)).
+- During when garbage collecting pods (`rkt gc`), all mounts will be umounted even when the pod dir is in inconsistent state ([#1828](https://github.com/coreos/rkt/pull/1828), [#1856](https://github.com/coreos/rkt/pull/1856))
 
-#### Other changes
+#### Improved documentation
 
-- Godeps: bump go-tspi to get license file ([#1855](https://github.com/coreos/rkt/pull/1855))
-- keystoretest: Move keygen.go to a separate directory ([#1848](https://github.com/coreos/rkt/pull/1848))
-- build: Check for patch and bc for kvm flavor ([#1860](https://github.com/coreos/rkt/pull/1860))
-- refactor image handling ([#1763](https://github.com/coreos/rkt/pull/1763))
-- bump appc to v0.7.4 and fix rkt code ([#1861](https://github.com/coreos/rkt/pull/1861))
-- build: Allow packagers to override stage1 version ([#1821](https://github.com/coreos/rkt/pull/1821))
+- Fix formatting, typos and outdated info in the documentation ([#1824](https://github.com/coreos/rkt/pull/1824)). The examples about rkt's configuration files are also clarified ([#1847](https://github.com/coreos/rkt/pull/1847)).
+- New documentation explaining [how cgroups are used by rkt](https://github.com/coreos/rkt/blob/master/Documentation/devel/cgroups.md) ([#1870](https://github.com/coreos/rkt/pull/1870)). This should make it easier for software developers to integrate rkt with monitoring software.
 
-#### Improved documentation
+#### API service
 
-- fix formatting, typos, outdated info ([#1824](https://github.com/coreos/rkt/pull/1824))
-- add paths to config examples ([#1847](https://github.com/coreos/rkt/pull/1847))
-- cgroup doc for developers ([#1870](https://github.com/coreos/rkt/pull/1870))
+The API service is meant to be used by orchestration tools like Kubernetes. The performances were improved by reducing the round-trips in the ListPods and ListImages requests ([#1786](https://github.com/coreos/rkt/pull/1786)). Those requests also gained multiple filters for a better flexibility ([#1853](https://github.com/coreos/rkt/pull/1853)).
```

Comment:
Maybe add a single bullet point for this paragraph, to be consistent with paragraphs above.